examples/ui-demo/src/hooks/useBreakpoint.tsx

```diff
@@ -16,10 +16,10 @@ type Breakpoint = keyof typeof breakpoints;
  *
  * @returns {Breakpoint} The current active breakpoint
  */
-export const useBreakpoint = (): Breakpoint => {
-  const [currentBreakpoint, setCurrentBreakpoint] = useState<Breakpoint>("sm");
+export const useBreakpoint = (): Breakpoint | undefined => {
```

Comment:
fix for modal popping up because on initial render, breakpoint was being set to `sm`